package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "postcss-input-range",
-  "version": "4.0.0",
+  "version": "5.0.0",
```

Comment:
Could you revert this ? It's taken care of by `npm version major` which will also tag

Reply:
Sure.

Reply:
Force pushed without the version bump :-)